src/System.Net.Security/src/System/Net/SecureProtocols/_SecuritySafeHandles.Windows.cs

```diff
@@ -402,7 +291,7 @@ internal sealed class SafeCredentialReference : CriticalHandleMinusOneIsInvalid
         //
         // Static cache will return the target handle if found the reference in the table.
         //
-        internal SafeFreeCredentials _Target;
+        internal SafeFreeCredentials _target;
```

Comment:
Seeing these is a bit concerning:
It appears that you have started work off an _old_ version of the code. This probably means that you've left out a lot of fixes that got in after the very thorough PR that @bartonjs helped with in the original System.Net.Security port.